src/main/java/org/codeforamerica/messaging/services/TemplateService.java

```diff
@@ -0,0 +1,47 @@
+package org.codeforamerica.messaging.services;
+
+import com.github.jknack.handlebars.Handlebars;
+import org.codeforamerica.messaging.models.Template;
+import org.codeforamerica.messaging.repositories.TemplateRepository;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class TemplateService {
+    private final TemplateRepository templateRepository;
+
+    public TemplateService(TemplateRepository templateRepository) {
+        this.templateRepository = templateRepository;
+    }
+
+    public String buildBodyFromTemplate(String templateName, Map<String, Object> templateParams) {
+        return buildContentFromTemplate(templateName, templateParams, Template::getBody);
```

Comment:
Why do we need the redirection?

Reply:
it's in anticipation of also having `buildSubjectFromTemplate`. I suppose the redirection isn't needed, but it hides the syntax of using `Template::getBody`.

Related to our standup discussion, there could also be `Template::getSmsBody` and `Template::getEmailBody`